crt_portal/cts_forms/tests/test_api.py

```diff
@@ -180,14 +180,14 @@ def test_response_detail_url(self):
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
         self.assertTrue("false" in str(response.content))
-        self.assertFalse("true" in str(response.content))
+        self.assertTrue("/api/responses" in str(response.content))
```

Comment:
Also, something I've seen throughout these files, that we can change as we see them - `assertContains` is a thing.

So, we can replace:

```
self.assertTrue(member in container)
```

with

```
self.assertContains(container, member)
```

The difference being that the latter will give a much better error message ("member" was not present in "container" - with their actual values) versus "false is not true"